docs/developers/full-stack-modular-development-guide.md

```diff
@@ -0,0 +1,712 @@
+    if (value === 'view-posts') {
+      fetchPosts()
+    }
+  }
+  
+  return (
+    <div style={outerContainerStyle}>
+      <div style={innerContainerStyle}>
+      <ConnectButton />
+      <div>
+        <button onClick={() => toggleView('view-posts')} style={buttonStyle}>View Posts</button>
+        <button  onClick={() => toggleView('create-post')} style={buttonStyle}>Create Post</button>
+      </div>
+      {
+        viewState === 'view-posts' && (
+          <div style={postContainerStyle}>
+            <h2>Posts</h2>
+            {
+              posts.map((post, index) => (
+                <div key={index}>
+                  <h4>{post.title}</h4>
+                  <ReactMarkdown>
+                   {post.postContent}
+                  </ReactMarkdown>
+                </div>
+              ))
+            }
+          </div>
+        )
+      }
+      {
+        viewState === 'create-post' && (
+          <div style={formContainerStyle}>
+              <h2>Create Post</h2>
+              <input
+                placeholder='Title'
+                onChange={e => setTitle(e.target.value)}
+                style={inputStyle}
+              />
+              <textarea
+                placeholder='Content'
+                onChange={e => setContent(e.target.value)}
+                style={inputStyle}
+              />
+              <button onClick={createPost}>Create Post</button>
+          </div>
+        )
+      }
+      </div>
+    </div>
+  )
+}
+
+const outerContainerStyle = {
+  width: '100vw',
+  height: '100vh',
+  padding: '50px 0px'
+}
+
+const innerContainerStyle = {
+  width: '900px',
+  margin: '0 auto',
+}
+
+const formContainerStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start'
+}
+
+const inputStyle = {
+  width: '400px',
+  marginBottom: '10px',
+  padding: '10px',
+}
+
+const postContainerStyle = {}
+
+const buttonStyle = {
+  marginTop: 15,
+  marginRight: 5,
+  border: '1px solid rgba(255, 255, 255, .2)'
+}
+
+export default App
+```
+
+## Testing it out locally
+
+Now we’re ready to run the app.
+
+Right now, the app is configured to be using `[localhost](http://localhost)` using the test account created by Foundry and Anvil.
+
+To use the test account, import the first private key given to you in the output when you ran the `anvil` command. This will be the account from which the program was deployed.
+
+Once you’ve imported the account, switch your wallet to `[localhost](http://localhost).`
+
+Next, run the React application:
+
+```bash
+npm run dev
+```
+
+## Testing it out on Troy
+
+Next, let’s run it on Troy testnet.
+
+To do so, first update the `contractAddress` variable with the contract address deployed to Troy:
+
+```jsx
+/* src/App.jsx */
+const contractAddress = "your-troy-contract-address"
+```
+
+Next, swap your wallet connect to the Troy testnet.
+
+When you run the app, you should now be connected to and using the Troy testnet. 
+
+## Next ste
```

Comment:
Haven't done a full review, just wanted to flag that this section seems incomplate atm